src/style.css

```diff
@@ -1 +1,80 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  min-width: 40px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav {
+  display: block;
+}
+
+.nav__list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  justify-self: start;
+  list-style: none;
+}
+
+.nav__link {
+  position: relative;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  display: flex;
+  align-items: center;
+  list-style: none;
+  flex-direction: row;
+  height: 60px;
```

Comment:
better use vertical paddings
what is the point do not hardcode height of header if we hardcode heigh of link then)